pkg/unittest/v1alpha3/default_machine_deployment.go

```diff
@@ -14,8 +14,8 @@ const (
 	DefaultMachineDeploymentID = "al9qy"
 )
 
-func DefaultMachineDeployment() apiv1alpha3.MachineDeployment {
-	cr := apiv1alpha3.MachineDeployment{
+func DefaultMachineDeployment() *apiv1alpha3.MachineDeployment {
```

Comment:
I had to change some parts in tests using pointer because otherwise metav1.Object would complain about  `implement "k8s.io/apimachinery/pkg/apis/meta/v1".Object (GetAnnotations method has pointer receiver)`